src/main/java/com/prototype/booking/api/referencedata/UserInfo.java

```diff
@@ -0,0 +1,20 @@
+package com.prototype.booking.api.referencedata;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Data
+@Entity
+public class UserInfo {
```

Comment:
Perhaps use either **User** or **Employee**, and change variables inside accordingly **empId** can be just **id**